indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -66,6 +67,12 @@ class AbstractReviewingState(TitledIntEnum):
     mixed = 5
 
 
+class EditTrackMode(int, IndicoEnum):
+    none = 0
```

Comment:
Yes, because the `_update_track` util is now used for *all* cases where we update one of the track lists